lib/payrix/resource/base.rb

```diff
@@ -27,7 +27,6 @@ def set(params)
              if @attrs.include? k
                public_send("#{k}=", v) if respond_to? "#{k}="
              else
-               self.class.send(:attr_accessor, k)
```

Comment:
After implementing the mixin, I noticed I still wasn't seeing the value be cast as an integer. I realized it was because of this line.

This method in the base class contains some magic, which when populated with data from the Payrix API, makes the data accessible through instance variables and attribute readers. This was overwriting the attribute reader defined by `.attr_cast_to_integer`. It is also not needed because we already define `.attr_accessor` for all attributes at the class level.